dockerfiles/SampleVcfToZarr/sample_vcf_to_zarr.py

```diff
@@ -132,9 +132,28 @@ def main():
     else:
         log_file = open(log, "w")
         log_file_needs_closing = True
-
+    
+    if not contigs:
+        with open(input_vcf_path) as vcf_open:
```

Comment:
If you wanted to support (b)gzip, you could probably do something like this (untested):

```
if input_vcf_path.endswith(".gz") or input_vcf_path.endswith(".bgz"):
    vcf_opener = gzip.open
else:
    vcf_opener = open
with vcf_opener(input_vcf_path, mode="rt") as vcf_open:
```